src/disputables/queries.js

```diff
@@ -1,9 +1,21 @@
 import gql from 'graphql-tag'
-import { GraphQLWrapper } from '@aragon/connect-thegraph'
+import { Client } from 'urql/'
```

Comment:
```suggestion
import { Client } from 'urql'
```

Not sure, but did we need the `/`?

Reply:
Oops, no we don't